### PR TITLE
Update to 3.9.2

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 3.9.2
- Patch pyproject.toml to correct metadata.

https://github.com/PyTables/PyTables/tree/v3.9.2
https://github.com/PyTables/PyTables/releases/tag/v3.9.2

### CODE REVIEW

File: recipe/patches/0002-no-python-blosc2-dep.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+diff --git a/pyproject.toml b/pyproject.toml
+index b503ab65..49d33800 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -72,7 +72,9 @@ dependencies = [
+     "numexpr >= 2.6.2",
+     "packaging",
+     "py-cpuinfo",
+-    "blosc2 >= 2.3.0",
```
Comment (lorepirri) [inline recipe/patches/0002-no-python-blosc2-dep.patch:9]: we could use `c-blosc2` in place of `blosc2`

```suggestion
+    "c-blosc2 >= 2.3.0",
```

**Review by lorepirri — APPROVED** [review]
LGTM

File: recipe/patches/0002-no-python-blosc2-dep.patch (hunk 1 above)
Comment (cbouss) [reply recipe/patches/0002-no-python-blosc2-dep.patch:9]: c-blosc2 is a shared library, I don't think the package has the required metadata for pip check.

Comment (lorepirri) [reply recipe/patches/0002-no-python-blosc2-dep.patch:9]: ah cool, then ignore my comment, thanks for the explanation

**Review by SC426 — APPROVED** [review]

**Pull request merged**